qcfractal/config.py

```diff
@@ -194,6 +202,13 @@ def database_uri(self, safe=True, database=None):
 
         return uri
 
+    @property
+    def view_path(self):
+        if self.view.directory is None:
+            return None
```

Comment:
What is the use case when the view directory is None?

Reply:
Previously, `None` meant disable views. I've added an `enable` flag in addition to the view path to make this more clear.